cmd/lib/spree_cmd/templates/extension/extension.gemspec

```diff
@@ -33,7 +33,7 @@ Gem::Specification.new do |s|
   s.add_development_dependency 'chromedriver-helper'
   s.add_development_dependency 'coffee-rails'
   s.add_development_dependency 'database_cleaner'
-  s.add_development_dependency 'factory_bot', '~> 4.7'
+  s.add_development_dependency 'factory_bot', '~> 5.0'
```

Comment:
@jkojro let's leave it 4.7 here as older Spree versions won't receive this update